bitnami/dokuwiki/README.md

```diff
@@ -82,6 +82,9 @@ The following table lists the configurable parameters of the DokuWiki chart and
 | `ingress.secrets[0].name`            | TLS Secret Name                                                                                       | `nil`                                                        |
 | `ingress.secrets[0].certificate`     | TLS Secret Certificate                                                                                | `nil`                                                        |
 | `ingress.secrets[0].key`             | TLS Secret Key                                                                                        | `nil`                                                        |
+| `securityContext.enabled`            | Enable securityContext on for Kibana deployment                                                       | `true`                                                       |
```

Comment:
AFAIK now we are trying to standardise this and split `securityContext` in two different ones:
* `podSecurityContext` (that should feature `fsGroup`)
* `containerSecurityContext` (that should feature `runAsUser`)
I think [redis-cluster](https://github.com/bitnami/charts/tree/master/bitnami/redis-cluster) is a good example of this approach. 😄

Also, I think you are referring to `Dokuwiki` instead of `Kibana`.